src/transport.cpp

```diff
@@ -69,6 +69,7 @@ namespace quicr {
       , logger_(spdlog::stderr_color_mt("MTC"))
       , server_config_({})
       , client_config_(cfg)
+      , tick_service_(std::make_shared<ThreadedTickService>())
```

Comment:
Should this be injected? Maybe this makes more sense, but wondering.

Reply:
I was thinking that too.  client and server have access to this so each doesn't really have to create it, but it might make sense to allow client/server do it and have transport use one passed in.